### PR TITLE
[ADD] Add partner_academic_title module

### PR DESCRIPTION
**Pull request by adrienpeiffer**
This module was written to adds possibility to define some academic titles and to add multiple titles on partners. there is a sequence on titles to display them in an order on the partner.

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:  (Code review + functional tests)

File: partner_academic_title/models/partner_academic_title.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class PartnerAcademicTitle(models.Model):
+    _name = 'partner.academic.title'
+
+    name = fields.Char(required=True)
```
Comment (dreispt) [inline partner_academic_title/models/partner_academic_title.py]: `translate=True`

Q: how do we add translation data for `partner_academic_title_data.xml`?

File: partner_academic_title/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        title_ids = self.academic_title_ids.sorted(lambda r: r.sequence)
+        for title in title_ids:
+            if display_title:
+                display_title = "%s %s %s" % (display_title, separator,
+                                              title.name)
+            else:
+                display_title = "%s" % (title.name)
+        self.academic_title_display = display_title
+
+    academic_title_ids = fields.Many2many(
```
Comment (dreispt) [inline partner_academic_title/models/res_partner.py:49]: wow! :-)

File: partner_academic_title/views/partner_academic_title_view.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_academic_title_form" model="ir.ui.view">
+            <field name="name">partner.academic.title.form</field>
+            <field name="model">partner.academic.title</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
```
Comment (dreispt) [inline partner_academic_title/views/partner_academic_title_view.xml:9]: Maybe a nitpick, but IMO `<sheet>` is to be used in "business documents" and not in code tables.

**Comment by lfreeke** [comment]
:+1:  (test)

**Comment by dreispt** [comment]
@adrienpeiffer Could you please make the partner.academic.title name translatable?

**Comment by adrienpeiffer** [comment]
Hi @dreispt ! It's done. Thanks !

**Comment by dreispt** [comment]
:+1:

**Comment by moylop260** [comment]
Could you fix
`partner_academic_title/models/res_partner.py:44:47: E127 continuation line over-indented for visual indent`?

**Comment by adrienpeiffer** [comment]
@moylop260 It's done ! Thanks !

**Pull request merged**